scripts/_common.sh

```diff
@@ -38,10 +38,6 @@ normalize-env-vars() {
   fi
 
   [ -n "$FOUNDRY_ETH_KEYSTORE_FILE" ] || die "$(err-msg-keystore-file)"
-
-  export FOUNDRY_ETHERSCAN_API_KEY="${FOUNDRY_ETHERSCAN_API_KEY:-$ETHERSCAN_API_KEY}"
-  # Some commands require the prefixed env var, while others require the unprefixed one.
-  export ETHERSCAN_API_KEY="$FOUNDRY_ETHERSCAN_API_KEY"
```

Comment:
Why remove this?

Reply:
After updating Foundry to the latest version it's won't work. They add some validation for Etherscan API key

Reply:
But does the source code verification work if we remove this?

Reply:
Yes, if you have `ETHERSCAN_API_KEY` set in env